### ISSUE TITLE
Vocabulary related issues

### ISSUE THREAD

**Issue by junkyul** [junkyul/allenel#4]
- pre-build vocabulary from training data 
when testing, what happens if a token is not in the vocab?
should be replaced to @@UNKNOWN@@ by indexer?

how this happens? does it touches existing vocabulary?
do I must turn off exend to false?

- use of pre-trained embedding like glove
how to use pre-trained word embedding together with data from instances?
simply turn on extend to true make things works?
what happens to the embedding (the dimension of the matrix is controlled by allennlp)

-unknown, padding? those tokens are in pre-trained embedding?

- advantage of pre-trained embeddings... earlier convergence?; if a special symbol is missing what happens? start maps to unknown end maps to unknown? when extend is off?

**Comment by junkyul** [comment]
some non english token examples in sentences.txt
Kraków
Ó
→
Fáil
षडानन
譚大仙
Pokémate
ポケメ
バトルシャーク
仁顯王后傳
인현왕후전
بيورو
انٹليجنس
